BlueprintUI/Sources/Extensions/UIAccessibilityTraits+Bueprint.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+
+extension UIAccessibilityTraits {
+
+    /// `UIAccessibilityTraits` contains a private trait that is used for back buttons
+    /// in `UINavigationController`, on `_UIButtonBarButton`.
+    /// We will use this same private trait to mirror the expected standard back button experience.
+    ///
+    /// ```
+    /// (lldb) po button
+    /// ▿ Optional<UIControl>
+    ///   - some : <_UIButtonBarButton: 0x1026336d0 ... >
+    ///
+    /// (lldb) p button?.accessibilityTraits
+    /// (UIAccessibilityTraits?) 134217728
+    ///
+    /// (lldb) po String(134217728, radix: 2)
+    /// "1000000000000000000000000000"
+    ///
+    /// (lldb) po String(134217728, radix: 2).count - 1
+    /// 27
+    /// ```
+    public static let backButton: UIAccessibilityTraits = UIAccessibilityTraits(rawValue: 1 << 27)
```

Comment:
Can we use a `package` ACL to avoid polluting the public namespace? I'm worried this will collide with a real trait in a future OS version.